src/zcbenchmarks.cpp

```diff
@@ -420,3 +455,14 @@ double benchmark_sendtoaddress(CAmount amount)
     return timer_stop(tv_start);
 }
 
+double benchmark_loadwallet()
+{
+    shutdown_wallet();
+    struct timeval tv_start;
+    bool fFirstRunRet=true;
+    timer_start(tv_start);
+    pwalletMain = new CWallet("wallet.dat");
+    DBErrors nLoadWalletRet = pwalletMain->LoadWallet(fFirstRunRet);
+    auto res = timer_stop(tv_start);
+    return res;
```

Comment:
Need to call `restart_wallet_after_load()` above here.

Reply:
Ohh right :) Fixed